beagle/src/main/kotlin/br/com/zup/beagle/android/utils/ToolbarManager.kt

```diff
@@ -228,13 +233,13 @@ internal class ToolbarManager(private val toolbarTextManager: ToolbarTextManager
         i: Int,
         rootView: RootView,
         container: BeagleFlexView,
-        screenComponent: ScreenComponent,
+        screen: Screen,
     ) {
         design?.let { designSystem ->
             items[i].image?.let { image ->
                 setShowAsAction(MenuItem.SHOW_AS_ACTION_ALWAYS)
```

Comment:
<table data-meta="generated_by_android">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle-android/blob/6567d57af6dc6216eb97d0ad3fe99ec417aa47c8/beagle/src/main/kotlin/br/com/zup/beagle/android/utils/ToolbarManager.kt#L240"></span>Prefer "<code>SHOW_AS_ACTION_IF_ROOM</code>" instead of "<code>SHOW_AS_ACTION_ALWAYS</code>"</td>
    </tr>
  </tbody>
</table>